tests/unit/util/parse_num_test.c

```diff
@@ -65,6 +66,7 @@ struct test_size {
         result = ~_t->output.result;                                                     \
                                                                                          \
         err = (*_func)(_t->input.str, _t->input.endptr ? &endptr : NULL, 0, 0, &result); \
+        merr_strinfo(err, buf, sizeof(buf), (merr_stringify)(err_ctx_strerror), NULL);   \
```

Comment:
Not sure what this does, is that a cast?  Why do you need to cast?

Reply:
`enum hse_err_ctx` != `int`, so the function pointers don't match

Reply:
Why not just use an int?

Reply:
I can do that